contracts/scripts/relay-skipped-tx.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable node/no-missing-import */
+import * as dotenv from "dotenv";
+
+import { ethers } from "hardhat";
+
+dotenv.config();
+
+async function main() {
+  const [deployer] = await ethers.getSigners();
+
+  const l1ScrollMessengerAddress = process.env.L1_SCROLL_MESSENGER_PROXY_ADDR!;
+  const l2EccContractAddress = process.env.L2_ECC_ADDR!;
+  const payload = process.env.SKIPPED_TX_PAYLOAD!; // TODO: calc the payload, parse as bytes
```

Comment:
can use `0x459be2920000000000000000000000000000000000000000000000000000000000000005` for payload

Reply:
```
import {
  getBytes,
} from "ethers";

...
const payloadStr = process.env.SKIPPED_TX_PAYLOAD!
const payload = getBytes(payloadStr);
...
```

Reply:
cast keccak "ecPairings(uint256)"